### ISSUE TITLE
Mini-batch and Multiple-gpu

### ISSUE THREAD

**Issue by DonghyunK** [wkentaro/pytorch-fcn#14]
Hi,

I want to use batch size more than 1.

So, I changed batch_size to 4. 
e.g.)
train_loader = torch.utils.data.DataLoader(
        torchfcn.datasets.SBDClassSeg(root, split='train', transform=True),
        batch_size=4, shuffle=True, **kwargs)

But, it raises an error so that I could not use it.

Could you please please let me know how to modify it?

and, is it possible to use multiple gpus by modifying codes? 

Thank you!

**Comment by wkentaro** [comment]
> So, I changed batch_size to 4.
e.g.)
train_loader = torch.utils.data.DataLoader(
torchfcn.datasets.SBDClassSeg(root, split='train', transform=True),
batch_size=4, shuffle=True, **kwargs)
But, it raises an error so that I could not use it.
Could you please please let me know how to modify it?

You need to use consistent image size.
See comment here: https://github.com/wkentaro/pytorch-fcn/issues/5#issuecomment-286032567

**Comment by wkentaro** [comment]
> and, is it possible to use multiple gpus by modifying codes?

It is possible, and I use multi-gpus in apc example: https://github.com/wkentaro/pytorch-fcn/blob/master/examples/apc/train_fcn32s.py#L60

**Comment by DonghyunK** [comment]
Thank you for your help!
